tests/cypress/integration/5-AI-SiteGen-onboarding-flow/7-editor.cy.js

```diff
@@ -282,4 +281,11 @@ describe.skip( 'SiteGen Site Editor Step', function () {
 			'.nfd-onboarding-sidebar--customize__design-colors-panel__custom__colors__container'
 		).should( 'not.exist' );
 	} );
+	
+	it( 'Check Save & Continue button functionality', () => {
+		cy.get('.nfd-onboarding-header--sitegen__editor__end__save-button__text', { timeout : 20000 } )
+			.click();
+		cy.get(`.${getAppId()}-app-container`, { timeout : 90000 } )
```

Comment:
Last doubt are the files linted just curious?

Reply:
Done